### PR TITLE
Fix for BatchNormalization layers with `center=False` or `scale=False`

### PR DESCRIPTION
**Pull request by jmduarte**
- Fix for batch normalization layers with `center=False` or `scale=False` (i.e. `beta` or `gamma` are `None`)

### CODE REVIEW

**Comment by jmitrevs** [comment]
What milestone is this for?

**Comment by jicampos** [comment]
@jmduarte I agree with @vloncar for the PyTorch parser. Would you like to add the fix for Keras now and follow up after #723?

**Comment by JanFSchulte** [comment]
I have updated #723 to support parsing of the `affine` argument so it will work with these changes.
